Overview.src.html

```diff
@@ -6,14 +6,14 @@
       Presentation API
     </title>
     <style>
-    @media print {
+@media print {
```

Comment:
I upgraded to HTML Tidy for Mac OS X version 4.9.35 (the latest tagged release). That seem to have fixed the issue.